core/src/main/java/org/geysermc/geyser/network/GameProtocol.java

```diff
@@ -49,9 +50,9 @@ public final class GameProtocol {
      * Default Bedrock codec that should act as a fallback. Should represent the latest available
      * release of the game that Geyser supports.
      */
-    public static final BedrockCodec DEFAULT_BEDROCK_CODEC = CodecProcessor.processCodec(Bedrock_v712.CODEC.toBuilder()
-        .minecraftVersion("1.21.20/1.21.21")
-        .build());
+    public static final BedrockCodec DEFAULT_BEDROCK_CODEC = CodecProcessor.processCodec(CodecProcessor.processCodec(Bedrock_v729.CODEC.toBuilder()
+            .minecraftVersion("1.21.30")
+            .build()));
```

Comment:
same indentation as the rest of the file. and the codec is being processed twice?